CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Coming soon
 - Enable adding additional user agent information to the HTTP requests made by the SDK.
 
+## [0.6.0] - 2020-05-28
+- Add support for Logs data type
```

Comment:
Any interest in adding weasel-words here like "beta" or "early", or "first-cut"?

Reply:
Sure, that's solid.  Will do.